stencil.config.ts

```diff
@@ -35,5 +44,11 @@ export const config: Config = {
       includePaths: ['./node_modules'],
       injectGlobalPaths: ["src/styles.scss"]
     })
-  ]
+  ],
+  //Uncomment to debug
+  // devServer: {
+  //   reloadStrategy: 'pageReload',
+  //   port: 3333,
+  //   address: 'http://192.168.0.11:3333'
```

Comment:
We should not include hard-coded IPs on public Repositories

Reply:
fixed!